diff --git a/supabase/functions/hello/index.ts b/supabase/functions/hello/index.ts
index ee0ff3617..f1e20b90e 100644
--- a/supabase/functions/hello/index.ts
+++ b/supabase/functions/hello/index.ts
@@ -2,8 +2,9 @@
 // https://deno.land/manual/getting_started/setup_your_environment
 // This enables autocomplete, go to definition, etc.
 
+import { serve } from "https://deno.land/std@0.177.1/http/server.ts"
 
-Deno.serve(async () => {
+serve(async () => {
   return new Response(
     `"Hello from Edge Functions!"`,
     { headers: { "Content-Type": "application/json" } },
